db/migrate/20210603094828_add_column_to_red_line_boundary_change_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToRedLineBoundaryChangeRequests < ActiveRecord::Migration[6.1]
+  def change
+    add_column :red_line_boundary_change_requests, :approved, :boolean
```

Comment:
given these are the last two migrations, we could just pop this into the create_red_line_boundary_change_requests and you can locally run rake db:migrate:redo